packages/react-native-gesture-handler/src/v3/hooks/callbacks/GestureHandlerEventWorkletHandler.ts

```diff
@@ -0,0 +1,40 @@
+import { CALLBACK_TYPE } from '../../../handlers/gestures/gesture';
+import { tagMessage } from '../../../utils';
+import { ReanimatedContext } from '../../../handlers/gestures/reanimatedWrapper';
+import { CallbackHandlers, UpdateEvent } from '../../types';
+import { isEventForHandlerWithTag, runCallback } from '../utils';
+
+export function getGestureHandlerEventWorkletHandler(
```

Comment:
I don't like `worklet` in the middle of event handler, I'd change it to either
```suggestion
export function getGestureHandlerWorkletEventHandler(
```
or
```suggestion
export function getGestureHandlerEventHandlerWorklet(
```